src/client/containers/RepoChangelog/index.js

```diff
@@ -0,0 +1,43 @@
+/* flow */
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { getChangelog } from 'ducks/repositories/selectors'
+import ChangesetList from 'components/ChangesetList'
+
+export type Props = {
+  params: Object,
+  branch: string,
+  repoName: string,
+  commits: Array<any>
+};
+
+class RepoChangelog extends Component {
+  // shouldComponentUpdate(nextProps, nextState) {
+  //   return nextProps.commits
+  // }
+
+  onSelectedChangesetsChange = (e, selectedChangesNumber) => {
+    this.setState({ enabled: selectedChangesNumber in [2, 1] })
```

Comment:
ok this is the feature you mentioned yesterday to create a PR from a range of changes, I guess we just have UI validation for now?